GuildedRoseLLC/Items/RemoteItemRepository.swift

```diff
@@ -2,12 +2,21 @@ import UIKit
 
 public class RemoteItemRepository: ItemRepository {
     
-    public init() {}
+    public static var stagingAPI: String = "https://floating-spire-59497.herokuapp.com/api/v1/items"
```

Comment:
I think this can be `private`. It might be worth looking through other parts of your application that are `public` when they don't need to be.